docs/use_cases/model_applications/s2s_mjo/UserScript_obsERA_obsOnly_RMM.py

```diff
@@ -1,61 +1,114 @@
 """
-UserScript: Make RMM plots from calculated MJO indices
-======================================================
+UserScript: Make ERA RMM plots from calculated MJO indices
+==========================================================
 
 model_applications/
```

Comment:
```suggestion
model_applications/s2s_mjo/UserScript_obsERA_obsOnly_RMM.py
```
The [template](https://github.com/dtcenter/METplus/blob/develop/docs/use_cases/use_case_documentation_template.py) has the full path on one line.